sb1-scope.json

```diff
@@ -119,25 +179,19 @@
         },
         {
           "enabled": true,
-          "file": "\\/.*\\/nettbank-privat\\/kort\\/rest\\/cards\\/credit\\/application.*",
-          "host": ".*\\.sparebank1\\.no$",
-          "protocol": "any"
-        },
-        {
-          "enabled": true,
-          "file": "\\/.*\\/nettbank-privat\\/kort\\/rest\\/cards\\/credit\\/limit.*",
+          "file": "\\/.*\\/nettbank-privat\\/kort\\/rest\\/cards\\/credit\\/.*",
```

Comment:
Spørsmål bare basert på path: Denne (og v2 under) er ikke for generelt?

Reply:
Dette er gamle APIer som vi er veldig nære å kunne skru av. Derfor tenker vi at det kan utelates fra scope.

Reply:
Den er god. Kan dere lage en PR for å fjerne det herfra også når ting skrus av?

Reply:
Det skal vi gjøre